spring-cloud-dataflow-admin-local/src/main/resources/admin.yml

```diff
@@ -2,7 +2,11 @@ server:
   port: 9393
 management:
   contextPath: /management
-
+info:
```

Comment:
ok, In that case, I think we need to have corresponding changes in all the SPI implementations so that we don't lose track.